mcs/class/corlib/Test/System.IO/DirectoryTest.cs

```diff
@@ -1113,7 +1109,7 @@ public void SetLastWriteTimeException2 ()
 	}
 	
 	[Test]
-	[ExpectedException(typeof(FileNotFoundException))]
+	[ExpectedException()]

```

Comment:
Why was the type removed?

Reply:
These now throw `DirectoryNotFoundException` on Unix, and `FileNotFoundException` on Windows